### PR TITLE
fix(deps): Add 'typesafe-dynamodb' as a dependency

### LINKED ISSUE #7: Add typesafe-dynamodb as a dependency
typesafe-dynamodb is a peer dependency of functionless. Our projen template should therefore add it as a dependency. 

__Originally posted by @sam-goodwin in https://github.com/functionless/functionless/issues/226#issuecomment-1152980494__

### PR DESCRIPTION
**Pull request by volkanunsal**
Fixes #7

### CODE REVIEW

**Comment by sam-goodwin** [comment]
Thanks for taking the initiative!

**Comment by sam-goodwin** [comment]
To fix the build, run `yarn build` locally and commit any changes.

**Comment by volkanunsal** [comment]
Fix the commit message and pushed it upstream.

**Comment by sam-goodwin** [comment]
> Fix the commit message and pushed it upstream.

Did you push it to the right branch? Diff looks unchanged.

**Comment by volkanunsal** [comment]
Yeah... `patch-1`. The original failure was because of the title of the PR. But I'm not sure what this one is about. 

```
Error: Files were changed during build (see build log). If this was triggered from a fork, you will need to update your branch.
```

**Comment by volkanunsal** [comment]
Ah, it looks like something is adding a comma.

**Comment by sam-goodwin** [comment]
> Ah, it looks like something is adding a comma.

Probably prettier! haha.

File: .vscode/settings.json [hunk 1]
```diff
@@ -22,7 +22,11 @@
   "[markdown]": {
     "editor.defaultFormatter": "esbenp.prettier-vscode",
     "editor.formatOnSave": true,
-    "editor.quickSuggestions": true,
+    "editor.quickSuggestions": {
+      "comments": "on",
+      "strings": "on",
+      "other": "on"
+    },
```
Comment (sam-goodwin) [inline .vscode/settings.json:29]: what does this change?

**Comment by sam-goodwin** [comment]
Thanks for the contribution, @volkanunsal! We are just now creating a discord community. If you want to be able to ask us questions more quickly and freely, feel free to come join https://discord.gg/zercqjtE

File: .vscode/settings.json (hunk 1 above)
Comment (volkanunsal) [reply .vscode/settings.json:29]: It must have been added automatically by VSCode.

**Pull request merged**